config/standalone/kv-cache-controller/patch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          args:
+            - --leader-elect
+            - --leader-election-id=aibrix-kv-cache-controller
+            - --health-probe-bind-address=:8081
+            - --metrics-bind-address=0
+            - --controllers=kv-cache-controller
```

Comment:
User deploys multiple controllers

ctrl-1: kv_cache enabled
ctrl-2: kv_cache + model_adapter enabled

In this scenario, can there be conflicts? or simply second one trying to write will just throw error

Reply:
technically, we do not have 2nd case, each controller only enable individual controller at this moment.

Reply:
user can install kv_cache or model_adapater separately. We may have an unified controller-manager for all of them later?